requirements.adoc

```diff
@@ -207,12 +269,30 @@ requested procedure for reporting and create the report.
 COMPONENTS::
 Its main components are
 
-* corresponding requested procedure
-* report status (claimed, completed, discontinued)
+* Radiology study / procedure
+* report status (claimed, ~~completed, discontinued~~) - preliminary , draft or finalized
 * report date
 * radiologist writing the report
 * report text
 
+```
+### Judy : Need a way to add an addendum to a report (if a mistake was made , or we have new images with new findings)
+The report requirements are described in detail in this document :::  https://docs.google.com/document/d/17u8kwEk-2D47ZsfSy_WM5LdpIuXwPm9LAQuoOSbQHHA/edit?usp=sharing
```

Comment:
the document is not very clean, hard to read. we should translate this into such a doc with clear stories, explanations.